src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -633,7 +631,7 @@ def __call__(self):
             if funcName == "repack":
                 self.handleRepackSettings()
 
-            if funcName in ["repack", "merge", "alcaHarvesting" ]:
+            if funcName in ["merge", "alcaHarvesting" ]:
```

Comment:
Ok, reviewed code, nothing to change. It overrides the right place, your code sees the changes value.